test/file.jl

```diff
@@ -1692,3 +1692,17 @@ end
         @test !isnothing(Base.Filesystem.getgroupname(s.gid))
     end
 end
+
+@testset "Disk space calculations work" begin
+    @test disk_total() == disk_total(pwd())
+    @test disk_used() == disk_used(pwd())
+    @test disk_available() == disk_available(pwd())
+
+    # Sanity check assuming disk is smaller than 16TB
+    TB = 2^40
+    @test disk_total() < 16TB
+
+    @test disk_used() < disk_total()
+    @test disk_available() < disk_total()
+    @test disk_used() + disk_available() == disk_total()
```

Comment:
Is it possible this can be flaky, if this changes while it's running? 🤔

Reply:
Yes, I think in principle it can. But the chances for that are pretty low (assuming Julia's buildfarm doesn't do heavy IO during tests), so I mused it might be fine. If we want to be on the safe side, we could turn this into a `<=`? Somewhat boring test then, but definitely safer.

Reply:
Someone's package runs this test on Julia PkgEval, and it fails frequently.

Reply:
Agreed that it'd become a somewhat boring test, but probably better to change it, yeah.

And in any case, it's still a somewhat silly test as-is, because you're just testing that `a + (b-a) = b` 😛

Reply:
done, thanks